paper.tex

```diff
@@ -707,17 +710,22 @@ \section*{Event vs Genome ARGs}\label{eARG}
 % section where we talk about stacking up recombination events
 % and their identifiability.]
 
-The eARG and gARG definitions provided here are apparently very
-similar, but, as we shall see, there are many advantages to decoupling
-our representation of patterns of genetic inheritance
-from the events that generated them.
-For simplicity, throughout the rest of the paper we will contrast the
-classical eARG with the gARG definition here.
-However, it is important to note that many of the benefits of the
-gARG encoding that we discuss are due to the encoding of inheritance via
-edges rather than nodes.
-Whether one regards nodes as genomes or events is essentially
-a matter of interpretation, and unimportant for many purposes.
+% The reader may arrive at the end of this section going, er, so what?
+% Try to tackle this head on by getting to the heart of the distinction.
+The difference between eARGs and gARGs are subtle, and whether
+we choose to interpret nodes as events or genomes
+is often of little practical importance.
+However, as we show in the remainder of this article,
+there are substantial practical gains
+from using the gARG encoding, allowing differential
+levels of information about the recombination process to be
+represented in a computionally efficient way.
+The key to this is not in how we interpret nodes, but in how
```

Comment:
"Not" is a little too strong, I think, as we do require nodes to be genomes if we are to condense multiple coalescences or recombinations into a single node (not allowed if the nodes are specific events). So perhaps something like "The key to this is less to do with the exact interpretation of nodes, and more to do with how we represent genetic inheritance. The critical point is that the edge annotation scheme used in the gARG encoding allows us to associate..."

Reply:
Good call, I'll update. I'll leave a pointer to the "Interpretation of Nodes" appendix also?